tests/fixture_test.go

```diff
@@ -64,6 +64,11 @@ func (fixture *Fixture) IncorrectAddressToVerify() *easypost.Address {
 		Country: "US",
 		Company: "EasyPost",
 		Phone:   "415-123-4567",
+		Verifications: &easypost.AddressVerifications{
+			Delivery: &easypost.AddressVerification{
+				Success: true,
```

Comment:
Why are we setting `Success` to `true` on the request data?

Reply:
this is for **IncorrectAddressToVerify** request data, I forgot to add this on the test suite PR.

Reply:
You are altering `response` details here, not `request` details. The proper way to pass this parameter is by adding `&CreateAddressOptions{Verify: []string{"delivery"}},` like the `CreateAddress` example shows in the inline comment right above it. 

It also appears there is a missing `e` in `Address` in that example, please also correct that.

Reply:
You are right, `CreateAndVerifyAddress` is missing a second parameter for verification.